packages/firestore/src/local/target_data.ts

```diff
@@ -104,7 +104,7 @@ export class TargetData {
       snapshotVersion,
       this.lastLimboFreeSnapshotVersion,
       resumeToken,
-      this.expectedCount
+      /** expectedCount= */ null
```

Comment:
nit: `/**` -> `/*`. The double-asterisk is used for method/class documentation. For a normal comment just use one asterisk.